pkg/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	separator = "_"
+
+	subsystem = "stonesoup_sprayproxy"
+
+	github                    = "github"
+	githubRequestsName        = subsystem + separator + github + separator + "requests_total"
+	forwarded                 = "forwarded"
+	forwardedRequestsName     = subsystem + separator + forwarded + separator + "requests_total"
```

Comment:
Similar to consideration for inbound requests, also include "http" modifier

```suggestion
	forwardedRequestsName     = subsystem + separator + "http_forwarded_requests_total"
```

Reply:
part of next push